src/cmd/src/cli/export.rs

```diff
@@ -202,10 +225,17 @@ impl Export {
                 };
                 t.push(value);
             }
-            result.push((t[0].clone(), t[1].clone(), t[2].clone()));
+            let table = (t[0].clone(), t[1].clone(), t[2].clone());
+            // Ignores the physical table
+            if !metric_physical_tables.contains(&table) {
+                result.push(table);
+            }
         }
+        let mut tables = Vec::with_capacity(metric_physical_tables.len() + result.len());
```

Comment:
Looks like we don't need to create a new Vector, just extend the `metric_physical_tables.extend(result)` and return it.

And it's better to rename the `result` to a meaningful name.

Reply:
The `metric_physical_tables` is a `HashSet`